pyrenew/latent/infectionswithfeedback.py

```diff
@@ -167,6 +167,10 @@ def sample(
                 **kwargs,
             )
         )
+
+        if inf_feedback_strength.ndim < Rt.ndim:
+            inf_feedback_strength = jnp.expand_dims(inf_feedback_strength, 0)
```

Comment:
Will this work if `Rt.ndim - inf_feedback_strength.ndim > 1`?

Reply:
work in the sense, these particular lines of code will just add a dimension to `inf_feedback_strength` along `axis=0`.
The code does not work and will raise an error because of this line further down
```
inf_feedback_strength.size != Rt.size
```

Reply:
In that case, I think the check here should just be 
```python
 if inf_feedback_strength.ndim ==  Rt.ndim -1:
```
so you can catch issues earlier.

Reply:
Good point! changed in https://github.com/CDCgov/PyRenew/pull/440/commits/cc61d0b2f3a973109808ffa2c222aaeb319be3c3

Reply:
I also made checks for shapes of `inf_feedback_strength` and `Rt` independent of other conditions!